File: 1. Introduction/max_value/max_value.js
const maxValue = (nums) => {
  
    // let maxValue = nums[0];
    // for(let index=1; index<nums.length; index++){
    //   maxValue = maxValue>nums[index]?maxValue:nums[index];
    // }
    
    // or
    let maxValue = -Infinity
    for(let num of nums){ // better readability
      if(num > maxValue){
        maxValue = num;
      }
    }
    
    return maxValue;
  };
  
  
  module.exports = {
    maxValue
  };

File: 1. Introduction/1. max_value/max_value.py
def max_value(nums):
  max_value = float("-inf")
  for num in nums:
    if num > max_value:
      max_value = num
  return max_value

print(max_value([4, 7, 2, 8, 10, 9]))
print(max_value([10, 5, 40, 40.3]))
print(max_value([-5, -2, -1, -11]))
print(max_value([42]))
print(max_value([1000, 8]))
print(max_value([1000, 8, 9000]))
print(max_value([2, 5, 1, 1, 4]))
